lib/applocale/config.rb

```diff
@@ -12,21 +12,21 @@ def self.load(config_file)
     def initialize(config_file)
       @config_file = config_file
       parse_config_file!
-      @api_key = env_or_config_value('api_key')
-      @project_id = env_or_config_value('project_id')
-      @locale_directory = env_or_config_value('locale_directory', default: 'config/locales')
+      @api_key = env_or_config_value("api_key")
+      @project_id = env_or_config_value("project_id")
+      @locale_directory = env_or_config_value("locale_directory", default: "config/locales")
     end
 
     def parse_config_file!
       validate_config_file!
 
       file_content = File.read(config_file_path)
       parsed_config_file = ERB.new(file_content).result
-      @configuration_data = YAML.load(parsed_config_file).dig("applocale") || {}
+      @configuration_data = YAML.safe_load(parsed_config_file).dig("applocale") || {}
```

Comment:
Trying to understand if we _have_ safe_load on YAML without requiring anything else. I don't see it in the Ruby docs 🤔

Reply:
```
➜ irb
irb(main):001:0> require "yaml"
=> true
irb(main):002:0> YAML.safe_load("")
=> nil
```

Reply:
Yeah, turns out it comes from Psych. Had to go dig in ruby. Epic.